features/admin.extensions.v1/configs/models/endpoints.ts

```diff
@@ -21,14 +21,21 @@
  */
 export interface ExtendedFeatureResourceEndpointsInterface {
     /**
-     * Remote user store agent connection API endpoint.
+     * On-prem user store agent connection API endpoint.
```

Comment:
on prem user store doesn't need an agent to connect, does it?
```suggestion
     * On-prem user store connection API endpoint.
```

Reply:
Agent connection is expected here.